### PR TITLE
Javascript utility

### PR DESCRIPTION
**Pull request by narasimha-1511**

### CODE REVIEW

File: javascript-utility/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import Popup from "./Popup";
+import Pop from "./Pop";
+import React from "react";
+import BhashiniTranslator from "@scaler-school-of-technology/bhashini-web-translator";
+
+const UrlParams = new URLSearchParams(document.currentScript.src.split("?")[1]);
+const APiKey = UrlParams.get("BHASHINI_API_KEY");
+const UserId = UrlParams.get("BHASHINI_USER_ID");
+let autoTranslate = localStorage.getItem("autoTranslate") || false;
+let tagretLanguage = localStorage.getItem("targetLanguage");
```
Comment (Diwakar-Gupta) [inline javascript-utility/src/App.jsx:10]: correct spelling here

File: javascript-utility/src/Popup.jsx [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+        transition: "transform 0.3s ease-in-out",
+        width: "280px",
+        height: "fit-content",
+        display: "flex",
+        flexDirection: "column",
+      }}
+    >
+      <img
+        src="img/scaler.png"
+        alt="Bhashini"
```
Comment (Diwakar-Gupta) [inline javascript-utility/src/Popup.jsx:70]: correct image link and alt for scaler and bhashini images

File: javascript-utility/src/main.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App";
+// import "./index.css";
+// import "./ppcss.css";
+document.body.innerHTML += '<div id="popup-container"></div>';
```
Comment (Diwakar-Gupta) [inline javascript-utility/src/main.jsx:6]: change id from `"popup-container"` to something containing `bhashini`.
there sould be other library that might contain `popup-container`.

**Comment by Diwakar-Gupta** [comment]
@nullHawk @narasimha-1511 

1. The button was not pressable on bottom half can only press on top. ( remove height css property from class="btn-container" )
2. rerender the whole component once translation is done this will fix the options valus in language selector.
3. We need to have a language selector for popup itself like we have planned for extension.

https://github.com/SST-Open-Source-Club/bhashini-web-translator/assets/39624018/e05bd13a-6bd5-433c-b511-e705f96c1656

**Comment by narasimha-1511** [comment]
@Diwakar-Gupta above errors are resolved and the popup selector is left

**Comment by Diwakar-Gupta** [comment]
@narasimha-1511 @nullHawk 
https://github.com/SST-Open-Source-Club/bhashini-web-translator/assets/39624018/d97b0226-5c17-42ee-bbfb-34f6336a444d

**Comment by narasimha-1511** [comment]
@Diwakar-Gupta the issue here was we assumed that the Bhashini translator  automatically recognizes the source language but its not recognizing it.

**Review by Diwakar-Gupta — APPROVED** [review]
lgtm

**Pull request merged**